src/components/Clamp.js

```diff
@@ -22,8 +22,9 @@ export default {
     },
     expanded: Boolean
   },
-  data () {
+  data ({$isServer}) {
     return {
+      isServer: $isServer,
```

Comment:
No need to add a data property here, we can access [`this.$isServer`](https://vuejs.org/v2/api/#vm-isServer) in `render`.